CONTRIBUTING.md

```diff
@@ -68,3 +70,11 @@ Where category is one of:
 
 `chore`: all changes to the repository that do not fit into any of the above categories.
 
+## Todo list
+
+HyperTheme Editor is in a `preview` state, it will be very helpful if you help with one of these tasks:
```

Comment:
it will be very helpful if you help -> it will be awesome if you want to help...